test/workspace/.vscode/settings.json

```diff
@@ -1,4 +1,19 @@
 // ! Only enable hybrid mode for testing, and should remove after `vue.volar` 3.0 release
 {
     "vue.server.hybridMode": true,
+    "prettify-ts.enabled": true,
+    "prettify-ts.typeIndentation": 4,
+    "prettify-ts.maxCharacters": 20000,
+    "prettify-ts.hidePrivateProperties": true,
+    "prettify-ts.maxDepth": 3,
+    "prettify-ts.maxProperties": 100,
+    "prettify-ts.maxSubProperties": 5,
+    "prettify-ts.maxUnionMembers": 15,
+    "prettify-ts.maxFunctionSignatures": 5,
+    "prettify-ts.skippedTypeNames": [],
+    "prettify-ts.unwrapArrays": true,
+    "prettify-ts.unwrapFunctions": true,
+    "prettify-ts.unwrapGenericArgumentsTypeNames": [
+        "Promise"
+    ],
```

Comment:
I'm a bit unsure about this addition, it was added every time I ran the tests. Ensure if we should check this in or not.